providers/curation/github.js

```diff
@@ -304,19 +327,21 @@ class GitHubCurationService {
     return coordinates.toString()
   }
 
-  _getBranchName(coordinates) {
-    const c = coordinates
-    return `${c.type.toLowerCase()}_${c.name}_${c.revision}_${moment().format('YYMMDD_HHmmss.SSS')}`
+  async _getBranchName(userGithub, serviceGithub) {
+    const committer = userGithub ? await userGithub.users.get({}) : await serviceGithub.users.get({})
```

Comment:
Probably should get this info once and stash it. We can do it in another PR but too many API calls to GitHub make them sad.

Reply:
I was able to cache userGithub.users.get({}); however, I am still unclear when to cache the serviceGithub.users.get({})